func_decor.py

```diff
@@ -0,0 +1,64 @@
+from time import time
+
+registry = {}
+
+
+def wrapper(func):
```

Comment:
usually we call wrapper internal function

Reply:
solved.